test/verify/check-machines-dbus

```diff
@@ -264,6 +264,18 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Attach some interface
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
+        # Change the os boot firmware configuration
+        if float(m.execute("virsh --version").strip()[:3]) >= 5.2:
+            b.wait_in_text("#vm-VmNotInstalled-firmware", "BIOS")
+            b.click("#vm-VmNotInstalled-firmware")
+            b.wait_present(".modal-body")
+            b.select_from_dropdown("select", "UEFI")
+            b.click("#firmware-dialog-apply")
+            b.wait_not_present(".modal-body")
+            b.wait_in_text("#vm-VmNotInstalled-firmware", "UEFI")
+        else:
+            b.wait_not_present("#vm-VmNotInstalled-firmware")
```

Comment:
Please add another test that (temporarily) moves  away the OVMF binary and validates that the selector is disabled with the expected tooltip.

Reply:
Added and cought an actual bug, fixed the related code path as well.